src/binding/c/init_api.txt

```diff
@@ -0,0 +1,106 @@
+# vim: set ft=c:
+
+MPI_Abort:
+    .desc: Terminates MPI execution environment
+    .skip: ThreadSafe, validate-ERROR_CODE
+    .extra: ignore_revoked_comm
+/*
+    Notes:
+    Terminates all MPI processes associated with the communicator 'comm'; in
+    most systems (all to date), terminates `all` processes.
+
+    .N NotThreadSafe
+    Because the 'MPI_Abort' routine is intended to ensure that an MPI
+    process (and possibly an entire job), it cannot wait for a thread to
+    release a lock or other mechanism for atomic access.
+*/
+
+MPI_Finalize:
+    .desc: Terminates MPI execution environment
+    .skip: ThreadSafe, global_cs
+
+MPI_Finalized:
+    .desc: Indicates whether 'MPI_Finalize' has been called.
+    .skip: initcheck
+    .docnotes: SignalSafe
```

Comment:
`.skip: global_cs`

Reply:
`.skip initcheck` implies `.skip: global_cs`, so I'll just delete `.docnots: SignalSafe` in this case.